src/Routes/ImageManager/Images.js

```diff
@@ -44,8 +52,28 @@ const CreateImageWizard = React.lazy(() =>
 
 const columns = ['Name', 'Version', 'RHEL', 'Type', 'Created', 'Status'];
 
+const defaultFilters = {
+  name: {
+    label: 'Name',
+    value: '',
+  },
+  distribution: {
+    label: 'Distribution',
+    value: [],
+  },
+  status: {
+    label: 'Status',
+    value: [],
+  },
+  imageType: {
+    label: 'Image type',
+    value: [],
+  },
+};
+
 const Images = () => {
   const history = useHistory();
+  const [activeFilters, setActiveFilters] = useState(defaultFilters);
```

Comment:
Since this is rather big object to change we should probably use [hooks-reference.html#usereducer](https://reactjs.org/docs/hooks-reference.html#usereducer)